mobify/platform-scaffold
# Fix initial crash on AMP dev server

```
    - Previously nodemon was used to watch a file that didn't exist
      until a webpack compile had succeeded.
    - Fix by starting webpack's watch process in a script and
      using the hooks it provides to start the AMP server in a
      subprocess.
    - Drops nodemon entirely.
```

## How to test-drive this PR
- Delete `amp/build`
- Run `npm run dev` and check it starts without errors.
- Check that changes to the app's source code restart the server.
